config/crd/common/bases/services.k8s.aws_adoptedresources.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.4.0
```

Comment:
How did this happen?

Reply:
guess this is the version of controller-gen I have installed locally...

Reply:
The build scripts should keep your kubebuilder inline with the version required. This seems like a mistake.

Reply:
Try to remove your local `bin` folder and run the `make` again